exopy_qm/tasks/tasks/ConfigureExecuteTask.py

```diff
@@ -0,0 +1,264 @@
+import importlib
+import importlib.util
+import ast
+
+from exopy.tasks.api import InstrumentTask
+from atom.api import Unicode, Long, Float, Value, Typed
+import sys
+import logging
+import time
+
+
+class ConfigureExecuteTask(InstrumentTask):
+    """Configures the QM, executes the QUA program and fetches the results
+
+    This task supports parameters in both the configuration and the
+    QUA program.
+
+    The program and config files are regular python file that should
+    contain at least two top-level functions:
+
+    - get_parameters() that should return the parameters dictionnary
+    of the file used to parametrize the config/program.
+
+    - get_config(parameters)/get_program(parameters) for the
+    configuration file and the program file respectively. The
+    parameters argument is a dictionnary containing the values entered
+    by the users and sould be converted to the appropriate python type
+    before using it.
+
+    The two files can be merged into one if wanted.
+
+    """
+
+    #: Path to the python configuration file
+    path_to_config_file = Unicode().tag(pref=True)
+
+    #: Module containing the configuration file
```

Comment:
I tend to prefer to see the private vars definition after the public methods and below a `# --- Private API ----` header.